features/step_definitions/category_steps.rb

```diff
@@ -0,0 +1,26 @@
+Given("the following categories exists") do |table|
+  table.hashes.each do |category|
+    create(:category, category)
+  end
+end
+
+Given("the following article are assigned to categories") do |table|
+  table.hashes.each do |article|
+    category = Category.find_by(name: article[:category])
+    article = Article.find_by(title: article[:title])
+    article.categories.push category
+  end
+end
+
+Given("I am on the Create Article page") do
+  visit new_article_path
+end
+
+Then("I select {string} from {string}") do |category, select_box|
+  select(category, from: select_box)
```

Comment:
I sense that you are mixing styles. Most of the step definitions omit the parathesis, here you have them. Stick to one style, please.